ledger/eval/applications.go

```diff
@@ -229,7 +229,7 @@ func (cs *roundCowState) NewBox(appIdx basics.AppIndex, key string, value []byte
 		return err
 	}
 	if exists {
-		return fmt.Errorf("attempt to recreate %s", key)
+		return fmt.Errorf("attempt to recreate %#x", key)
```

Comment:
While we're in here, let's add the word "box" after "recreate" to match the wording elsewhere.

Reply:
yep, will do